tests/test_svg.py

```diff
@@ -130,273 +130,19 @@ def test_diagram_saves(self, tmp_svg: SVGDiagram) -> None:
-                "fill": "#4A4A97",
-            },
-            ".LogicalArchitectureBlank g.Box.LogicalFunction > line": {
-                "stroke": "#095C2E",
-            },
-            ".LogicalArchitectureBlank g.Box.LogicalFunction > rect,"
-            " .LogicalArchitectureBlank g.Box.LogicalFunction > use": {
-                "fill": "#C5FFA6",
-                "stroke": "#095C2E",
-            },
-            ".LogicalArchitectureBlank g.Box.LogicalFunction > text": {
-                "fill": "#095C2E",
-            },
-            ".LogicalArchitectureBlank g.Edge.FunctionalExchange > rect": {
-                "fill": "none",
-                "stroke": "none",
-            },
-            ".LogicalArchitectureBlank g.Circle.FunctionalExchange > circle": {
-                "fill": "#095C2E",
-                "stroke": "none",
-            },
-            ".LogicalArchitectureBlank g.Edge.FunctionalExchange > path": {
-                "stroke": "#095C2E",
-            },
-            ".LogicalArchitectureBlank g.Edge.FunctionalExchange > text": {
-                "fill": "#095C2E",
-            },
-            ".LogicalArchitectureBlank g.Edge.ComponentExchange > rect": {
-                "fill": "none",
-                "stroke": "none",
-            },
-            ".LogicalArchitectureBlank g.Circle.ComponentExchange > circle": {
-                "fill": "#4A4A97",
-                "stroke": "none",
-            },
-            ".LogicalArchitectureBlank g.Edge.ComponentExchange > path": {
-                "stroke": "#4A4A97",
-            },
-            ".LogicalArchitectureBlank g.Edge.ComponentExchange > text": {
-                "fill": "#4A4A97",
-            },
-            ".LogicalArchitectureBlank g.Edge.FIPAllocation > rect": {
-                "fill": "none",
-                "stroke": "none",
-            },
-            ".LogicalArchitectureBlank g.Circle.FIPAllocation > circle": {
-                "fill": "#E08503",
-                "stroke": "none",
-            },
-            ".LogicalArchitectureBlank g.Edge.FIPAllocation > path": {
-                "stroke": "#E08503",
-            },
-            ".LogicalArchitectureBlank g.Edge.FOPAllocation > rect": {
-                "fill": "none",
-                "stroke": "none",
-            },
-            ".LogicalArchitectureBlank g.Circle.FOPAllocation > circle": {
-                "fill": "#095C2E",
-                "stroke": "none",
-            },
-            ".LogicalArchitectureBlank g.Edge.FOPAllocation > path": {
-                "stroke": "#095C2E",
-            },
-        }
-
+    def test_base_css_styles(self, tmp_json: pathlib.Path) -> None:
         tree = etree.fromstring(
             SVGDiagram.from_json_path(tmp_json).to_string()
         )
-        style_ = tree.xpath(
-            "/x:svg/x:defs/x:style",
-            namespaces={"x": "http://www.w3.org/2000/svg"},
-        )[0]
-
-        stylesheet = cssutils.parseString(style_.text)
```

Comment:
These tests were the only place where we used `cssutils` - now that we don't have a stylesheet to test anymore, we can get rid of that dependency entirely.